include/deal.II/base/tensor.h

```diff
@@ -2676,7 +2676,9 @@ constexpr inline DEAL_II_ALWAYS_INLINE
 }
 
 
-/** @} */
+/**
+ * @}
+ */
```

Comment:
For some reason, it didn't want to understand the grouping with the former form. It does with the latter.